### ISSUE TITLE
`helmfile test` does not use `helmDefaults.timeout` value

### ISSUE THREAD

**Issue by davidnortonjr-sps** [roboll/helmfile#1191]
I have the following in my helmfile.yaml:

```
helmDefaults:
  tillerNamespace: foobar
  wait: true
  timeout: 900
  force: true
  verify: false

releases:
...
```

When I run ```helmfile --log-level debug test```, I see:

```
exec: helm test apigw --timeout 300 --tiller-namespace foobar
```

It seems as if it should use the `helmDefaults.timeout` value -- or at least a new value to allow us to specify the test timeout value as a first-class citizen in helmfile.yaml.

**Comment by RaymondKYLiu** [comment]
Hi, @davidnortonjr-sps 
Try to use  `helmfile test --args '--timeout 900'`, this works to me.

**Comment by davidnortonjr-sps** [comment]
Thanks @RaymondKYLiu , that works (actually I can even specify directly, e.g. `helmfile test --timeout 900`). However, I was looking for a YAML option so my company's users can specify the flag in source control.

**Comment by mumoshu** [comment]
This is definitely a bug. And this should be fixed in #1211

**Comment by davidnortonjr-sps** [comment]
thank you so much @mumoshu!
